Jenkinsfile

```
@@ -48,6 +49,79 @@ pipeline {
 
     stage('Tests') {
       parallel {
+        stage('Prepare Windows 2016 with containers') {
+          agent { label 'executor-windows-2016-containers' }
+          stages {
+            stage('Configure Windows Node'){
+              steps {
+                powershell '.\\expose-daemon.ps1'
+                script {
+                  env.WINDOWS_IP = powershell(returnStdout: true, script:  '(curl http://169.254.169.254/latest/meta-data/local-ipv4).Content').trim()
+                  env.WINDOWS_DOCKER_CERT_CA = powershell(returnStdout: true, script:  'cat $env:USERPROFILE\\.docker\\ca.pem')
+                  env.WINDOWS_DOCKER_CERT_CERT = powershell(returnStdout: true, script:  'cat $env:USERPROFILE\\.docker\\cert.pem')
+                  env.WINDOWS_DOCKER_CERT_KEY = powershell(returnStdout: true, script:  'cat $env:USERPROFILE\\.docker\\key.pem')
+                  env.WINDOWS_READY = true
+                }
+              }
+            }
+            stage('Wait for Main Node') {
+              steps {
+                waitUntil {
+                  script {
+                    return (env.MAIN_NODE_DONE == "true");
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        stage('With Windows 2016 containers') {
```

Maybe trim this down? Thoughts?
```suggestion
        stage('Win2016') {
```